docs/build/circuit-construction.ipynb

```diff
@@ -0,0 +1,486 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "bc54d7bc-b2d8-4b30-9b0b-05689e07a463",
+   "metadata": {},
+   "source": [
+    "# Construct a circuit"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c50d8e43-ae82-4e41-8d17-a37332d1bf6d",
+   "metadata": {},
+   "source": [
+    "This page takes a closer look at Qiskit's [`QuantumCircuit`](/api/qiskit/qiskit.circuit.QuantumCircuit) class, including some more advanced methods you can use to create quantum circuits."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2664d407-aa95-43a3-9101-d3ad58c2df58",
+   "metadata": {},
+   "source": [
+    "## What is a quantum circuit?\n",
+    "\n",
+    "A simple quantum circuit is just a collection of qubits, plus a list of instructions that act on those qubits. To demonstrate, the following cell creates a new circuit with two new qubits, then displays the circuit's [`qubits`](/api/qiskit/qiskit.circuit.QuantumCircuit#qubits) attribute."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "b410d397-b67d-4f31-90cf-9c1c34c157c5",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[Qubit(QuantumRegister(2, 'q'), 0), Qubit(QuantumRegister(2, 'q'), 1)]"
+      ]
+     },
+     "execution_count": 1,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from qiskit import QuantumCircuit\n",
+    "\n",
+    "qc = QuantumCircuit(2)\n",
+    "qc.qubits"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f5c95cb2-a94f-48f3-b2a6-8ec6c25da5cd",
+   "metadata": {},
+   "source": [
+    "When we add an instruction to the circuit, this appends the instruction to the circuit's [`data`](/api/qiskit/qiskit.circuit.QuantumCircuit#data) attribute. The following cell output shows `data` is a list of [`CircuitInstruction`](/api/qiskit/qiskit.circuit.CircuitInstruction) objects, each of which has an `operation` attribute, and `qubits` attribute."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "f7b5573c-b2b2-4cbf-ba55-c53c9221ce71",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[CircuitInstruction(operation=Instruction(name='x', num_qubits=1, num_clbits=0, params=[]), qubits=(Qubit(QuantumRegister(2, 'q'), 0),), clbits=())]"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "qc.x(0)  # Add X-gate to qubit 0\n",
+    "qc.data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "17a82d8a-b717-44b8-b3f8-ce89e2588261",
+   "metadata": {},
+   "source": [
+    "The easiest way to view this information is through the [`draw`](/api/qiskit/qiskit.circuit.QuantumCircuit#draw) method, which returns a visualization of a circuit."
```

Comment:
At this point you might want to mention there are different ways to visualise circuits and link them to https://docs.quantum-computing.ibm.com/build/circuit-visualization